### ISSUE TITLE
prepend/append -> wrong path -> no error?

### ISSUE THREAD

**Issue by thomasaull** [bdkjones/CodeKit#326]
When combining Javascript files with Codekits prepend/append comments and there's a wrong file path (file does not exist) there should be a warning about CodeKit not being able to include this file.

Steps to reproduce:
Use Codekits @prepend with a path to a file which does not exist on your hard drive

**Comment by bdkjones** [comment]
Agreed. It's on my list to implement, it's just complex for a variety of reasons.

**Comment by thomasaull** [comment]
Good to know it's on your list ;)
